requirements.txt

```diff
@@ -1,3 +1,4 @@
+# This file is automatically generated by GitHub Actions and contains pinned versions for all transitive Python dependencies. Do not modify this file!
```

Comment:
How does this get generated?  Is it manually put in here?

Reply:
It is generated here: https://github.com/stanford-crfm/helm/blob/dcc6190e9b0640dc39dea41b8027a111346c6ae5/.github/workflows/update-requirements.yml#L48-L49